Noborder

strobelm opened the pull request:
This should resolve #626 and  #437 

I'm not sure if 
```
var color = (el.noborder.value === true || el.border.value === false) ? col : Render2D.makeColor(defaultAppearance.textColor, el.alpha.value);
``` 
is the most elegant solution, i would appreciate input on that.

kranich commented:
I personally do not think that this is elegant. I find this less readable than what you have implemented at the moment. I would rather leave it as it is, spread over several lines, unless there is a significant performance penalty for that.

kranich commented:
Just my humble opinion, feel free to ignore :-)

strobelm commented:
Sorry the comment was a bit misleading. It wasn't about the code itself
but rather about the basic idea of implementation.

The code had a little in the alpha handling which i already fixed and en
passant i refactored the code a little bit.

Am 19.09.2017 um 13:03 schrieb Stefan Kranich:
> I personally do not think that this is elegant. I find this less
> readable than what you have implemented at the moment. I would rather
> leave it as it is, spread over several lines, unless there is a
> significant performance penalty for that.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CindyJS/CindyJS/pull/628#issuecomment-330505340>, or
> mute the thread
> <https://github.com/notifications/unsubscribe-auth/ANhz4l-0sPnFTqBoExTUpuWdTPN4tbt8ks5sj5-WgaJpZM4Pa25z>.
> 

-- 
Michael Strobel, M.Sc.

Technische Universität München -- Zentrum Mathematik
Lehrstuhl für Geometrie und Visualisierung (M10)
Boltzmannstr. 3, 85747 Garching bei München
https://www-m10.ma.tum.de/bin/view/Lehrstuhl/MichaelStrobel

kranich commented:
I think the approach is elegant enough.

montaga commented:
To me, everything also looks good.
